core/blockchain.go

```diff
@@ -1213,3 +1213,6 @@ func (self *BlockChain) GetBlockHashesFromHash(hash common.Hash, max uint64) []c
 func (self *BlockChain) GetHeaderByNumber(number uint64) *types.Header {
 	return self.hc.GetHeaderByNumber(number)
 }
+
+// Accessor function to return the blockchain's ChainConfig
+func (self *BlockChain) ChainConfig() *ChainConfig { return self.chainConfig }
```

Comment:
Oops. Teaches me to try and do "trivial" fixes without proper development practices. Done.